src/modules/endpoint/sagas/watchFetchCategory.js

```diff
@@ -62,26 +62,6 @@ function * fetchCategory (dataContainer: DataContainer, action: FetchCategoryAct
   }
 }
 
-function * cancelableFetchCategory (dataContainer: DataContainer, action: FetchCategoryActionType): Saga<void> {
-  const { cancel } = yield race({
-    response: call(fetchCategory, dataContainer, action),
-    cancel: take('SWITCH_CONTENT_LANGUAGE')
-  })
-
-  if (cancel) {
-    const newLanguage = cancel.params.newLanguage
-    const newFetchCategory: FetchCategoryActionType = {
-      type: 'FETCH_CATEGORY',
-      params: {
-        language: newLanguage,
-        path: `/${action.params.city}/${newLanguage}`,
-        ...action.params
-      }
-    }
-    yield put(newFetchCategory)
-  }
-}
```

Comment:
Why did you remove this?

Reply:
Changing language very fast is indeed possible on my emulator :thinking: I think this was the very reason why this code was introduced.

Reply:
Did you check if this is still a problem?